python/py-ds-store/Portfile

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        dmgbuild ds_store 1.3.1 v
+
+name                py-ds-store
+
+categories-append   devel
+license             {MIT PSF}
+platforms           {darwin any}
+supported_archs     noarch
+maintainers         nomaintainer
+description         Manipulate Finder .DS_Store files from Python
+long_description    ds_store lets you examine and modify .DS_Store files from Python code\; since it is written in pure Python, it is portable and will run on any platform, not just Mac OS X.
+
+homepage            https://ds-store.readthedocs.io/en/latest/
+
+checksums           md5     2259e8cc693915eebf2aac1d43ebdd99 \
+                    rmd160  38d637cd6b1f203f75710026323a32141f99fac2 \
+                    sha256  00cf7d692829793a50e3f56660fcdda76988088ae319ad8db86330baeadded46 \
+                    size    24832
+
+python.versions     38 39 310 311 312
```

Comment:
can we only add recent Python versions?

Reply:
I'll change it to 3.9+.
I'm not usually a fan of bending over backwards to support dead software, but in this case it "just works"